CHANGELOG.md

```diff
@@ -1,3 +1,9 @@
+## 0.28.4 (2017-10-25)
+
+#### New Features
+
+* `exo-deploy`: allow traffic from bastion instances to RDS secruity groups
```

Comment:
Were you able to test this with a deploy of EDX and be able to connect? Also thinking back on the implementation the 5432 port is specific to Postgres. Mysql for instance uses a different port but would also use RDS

Reply:
Good point. For this release I'll note that it only allows postgres traffic. I did a test deploy and let it go until the plan step. Then I read the plan to verify that the sg was being modified, but I didn't actually apply the deploy. Should I test that right now?

@alexdavid should we allow all traffic from bastion for each of the respective db ports?

Reply:
I don't think we necessarily need to support the other versions at the moment but at least build this in a way that we can add support for the order dbs down the line

Reply:
Yes it's certainly built in a way to allow support for other dbs, even right now when we're only supporting postgres

Reply:
Shouldn't the port be a variable though? Where 5432 is defined once as part of the postgres dependency and we would have a different one for mysql

Reply:
Good point, didn't think about that. We honestly could just open up all communication to the bastion instance like we do for the ECS cluster.

Reply:
I like the simpler approach